fr32/Cargo.toml

```diff
@@ -26,8 +25,8 @@ rand_xorshift = "0.3"
 
 [features]
 default = []
-cuda = ["bellperson/cuda"]
-opencl = ["bellperson/opencl"]
+cuda = []
+opencl = []
```

Comment:
You can remove the whole features section.

Reply:
ok.